.github/main.workflow

```diff
@@ -0,0 +1,9 @@
+workflow "API extractor" {
+  resolves = ["Breaking change detection bot"]
+  on = "pull_request"
```

Comment:
Looking at the event reference doc,
https://developer.github.com/actions/managing-workflows/workflow-configuration-options/#events-supported-in-workflow-files
it seems like `on = "pull_request"` might trigger the breaking change bot for much more events than we would like.  Would using `on = "push"` work better in this case, the goal being we want the bot to run for any new code pushed to a PR ( or upon the PR creation )?

What do you think?

Reply:
Ok. I was worried about pushes without PR and comment then would fail, but I would look more into it. Then we would avoid all those builds after label changes and similar. (My wish for github is to provide option to specify on with subevents like `on = "pull_request.synchronize")

Reply:
Changed to `push` event :)